examples/beam/src/main/java/edu/snu/nemo/examples/beam/GenericSourceSink.java

```diff
@@ -94,7 +94,10 @@ public static PDone write(final PCollection<String> dataToWrite,
       dataToWrite.apply(ParDo.of(new HDFSWrite(path)));
       return PDone.in(dataToWrite.getPipeline());
     } else {
-      return dataToWrite.apply(TextIO.write().to(path));
+      // Added withWindowedWrites() to local file writes. This is necessary for FileResult coders.
+      // If not specified, FileResultCoder#encode will be blocked. See windowCoder in FileResultCoder#encode
```

Comment:
Can you elaborate on what 'blocked' means?
Does this mean compile-time translation failure, or run-time execution failure?

Reply:
Since this problem is caused from Beam, It was difficult to fully specify the cause. The execution is blocked at [here](https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java#L1141) when trying to encode FileResult.